src/stories/button/button.stories.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Button } from '../../components';
+
+const stories = {
+	title: 'Button',
+	component: Button,
```

Comment:
In storybook the components are always returned by means of a questionnaire (and title : Components/Button)

Reply:
After reflexion, Button is just an utility component. We can't display a component Button alone (it came with Roster, Missing etc ..) so i delete this stories